### ISSUE TITLE
Pharo 64-bit image crashes when trying to push changes

### ISSUE THREAD

**Issue by olekscode** [pharo-vcs/iceberg#425]
**OS:** Linux Mint 18.2 Sonya
**Image:** Pharo6.1-64.image, Latest update: #60510
**VM:**  201707201942, Date: Thu Jul 20 12:42:21 2017
Both Image and VM were acquired in a single archive from [http://pharo.org/download](http://pharo.org/download)

**Steps to reproduce the bug**

1.  Load the project using the following Metacello script
``` Smalltalk
Metacello new
  baseline: 'DataFrame';
  repository: 'github://PolyMathOrg/DataFrame';
  load.
```
2. Make some changes inside the packages of the project. Save the image
3. Press "Clone repository" in Iceberg, past the SSH: `git@github.com:PolyMathOrg/DataFrame.git`, and press "Create repository".
4. Write a commit messge and try to push it - you'll get a crash.
5. Now you are back to your last save. So you start at step 3. But the commit in your local repository was already made, so just open "New local commits to be published" and try to push it (by clicking on it). The moment you click, it will crash again

**Comment by estebanlm** [comment]
(NOTE TO SELF) this is in 6.1, I would like to test on 7.0.
I'm guessing it has something to do with structure initialisation (since 6.1 is not bootstrapped but converted).

**Comment by estebanlm** [comment]
fixed on 0.5.7

**Comment by estebanlm** [comment]
bah... at least it *should* be fixed. Can you check and re-open if not?
